test/integration/dss/test_dss_api.py

```diff
@@ -4,36 +4,36 @@
 from concurrent.futures import ThreadPoolExecutor
 import datetime
 import filecmp
-from hca.util import tsv
+from dbio.util import tsv
 import itertools
 import os
 import sys
 import tempfile
 import uuid
 import unittest
-from fnmatch import fnmatchcase
+from fnmatch import fnmatcdbiose
```

Comment:
i think this modification might have been from matching `*hca*` characters...... 🤔

Reply:
Yup, good call

Reply:
Fixed in 464bcc2